_layouts/tags.html

```diff
@@ -3,7 +3,7 @@
 # All the Tags of posts.
 ---
 
-<div id="tags" class="d-flex flex-wrap mx-xl-2">
+<div id="tags" class="d-flex flex-wrap mx-xl-2 mb-5">
```

Comment:
Was this intentional?

Reply:
Yes, when the number of tags is too large and exceeds the height of the screen, a margin needs to be added at the bottom.

Example for 1000 tags:

![Screen Shot 2023-06-19 at 13 13 30](https://github.com/cotes2020/jekyll-theme-chirpy/assets/11371340/7f665827-781e-4e11-9b9b-48ff5299b323)